### PR TITLE
update perftest variables

### PR DESCRIPTION
**Pull request by naikajah**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: variablegroups/PERTTEST-AKS-Common.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      "value": "10.10.44.0/23"
+    },
+    "Subscription.AKS.vmSize": {
+      "value": "Standard_D4s_v3"
+    },
+    "Subscription.AKS.VNet.Cidr": {
+      "value": "10.10.40.0/21"
+    },
+    "Subscription.AKS.VNet.DnsServers": {
+      "value": "[\"10.96.139.254\"]"
```
Comment (adusumillipraveen) [inline variablegroups/PERTTEST-AKS-Common.json]: This one seems to be of AAT ? Not sure if we use same for perftest , worth checking.

Comment (naikajah) [reply variablegroups/PERTTEST-AKS-Common.json]: yeah updated the one for perftest.

File: variablegroups/PERTTEST-AKS-Common.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+  "variables": {
+    "Env.Core.ResourceGroup": {
+      "value": "core-infra-$(Env.Override.Name)-rg"
+    },
+    "Env.Override.Name": {
+      "value": "perftest"
```
Comment (timja) [inline variablegroups/PERTTEST-AKS-Common.json]: isn't this test?
```suggestion
      "value": "test"
```

Comment (naikajah) [reply variablegroups/PERTTEST-AKS-Common.json]: aah yes good point.

**Pull request merged**